index.html

```diff
@@ -58,6 +59,216 @@ <h1 class="identity_text">I'm Simba, I'm glad to see you!</h1>
+        </li>
+        <li class="technology">
+          <p class="technology_text">css</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">javascript</p>
+        </li>
+      </ul>
+      <button class="project_button" type="submit">See Project</button>
+    </div>
+
+    <div class="project">
+      <img src="Snapshoot_Portfolio3.png" class="project_img" alt="project_image">
+      <h2 class="project_title">Multi-Post Stories</h2>
+      <ul class="project_details">
+        <li class="list_items">
+          <p class="list_items_text">CANOPY</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">Back End Dev</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">2015</p>
+        </li>
+      </ul>
+      <p class="project_description">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+      <ul class="technology_used">
+        <li class="technology">
+          <p class="technology_text">html</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">css</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">javascript</p>
+        </li>
+      </ul>
+      <button class="project_button" type="submit">See Project</button>
+    </div>
+  </section>
+
+  <section class="aboutme_container">
+    <div class="aboutme_class">
+      <h2 class="identity_text">About Myself</h2>
+      <p class="description_text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <p class="connect_text">LET'S CONNECT</p>
+      
+      <ul class="social_media_icons">
+        <li>
+          <a>
+          <img src="twitter.png" class="icons" alt="twitter">
+          </a>
+        </li>
+        <li>
+          <a>
+          <img src="github.png" class="icons" alt="github">
+        </a>
+        </li>
+        <li>
+          <a>
+          <img src="Linkedin_icon.png" class="icons" alt="linkedin">
+          </a>
+        </li>
+        <li>
+          <a>
+          <img src="medium.png" class="icons" alt="medium">
+          </a>
+        </li>
+        <li>
+          <a>
+          <img src="angellist.png" class="icons" alt="angellist">
+          </a>
+        </li>
+      </ul>
+      <button class="resume_button" type="submit">Get my resume</button>
+      <ul class="technology_and_skills">
+        <li class="technology_and_skills_item">
+          <h2 class="technology_and_skills_text">Languages</h2>
```

Comment:
- [ ] Please you still missing what the previous review said: Kindly use the same font family as in the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
- [ ] From Figma: 

![m1](https://user-images.githubusercontent.com/63972670/159114003-b067698d-75c2-41fb-a2d7-3ccd66e6e713.JPG)

- [ ] From your design:

![m2](https://user-images.githubusercontent.com/63972670/159114044-a7440837-b117-49f9-8b03-c88a234599a2.JPG)